app/services/weet.js

```diff
@@ -0,0 +1,23 @@
+const { default: Axios } = require('axios');
```

Comment:
File names must be snake case and plural, see Wolox NodeJS Standards and best practices https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md

Reply:
You can also delete the .keep file from the services folder